### PR TITLE
Reorganize Configuration section

### PR DESCRIPTION
**Pull request by janan07**

### CODE REVIEW

**Review by samanthasusu — APPROVED** [review]
LGTM! Thank you!

File: docs/user-guide/configuring-overview.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+To cofigure Zowe runtime, choose from the following options:
+
+* Option 1: Configure Zowe manually using the `zwe init` command group
+* Option 2: Configure Zowe with z/OSMF workflows
+
+**Option 1**: To run the `zwe init` command, it is necessary to create a Zowe configuration file. For more information about this file, see [Zowe configuration file](./installandconfig/#zowe-configuration-file). 
+
+Once your configuration file is prepared, see [Initializing the z/OS system with `zwe init`](./initialize-zos-system.md), for more information about using the `zwe init` command group.
+
+**Option 2**: You can execute the Zowe configuration workflow either from a PSWI during deployment, or later from a created software instance in z/OSMF. Alternatively, you can execute the configuration workflow z/OSMF during the workflow registration process.
```
Comment (skurnevich) [inline docs/user-guide/configuring-overview.md:30]: I would merge these in just two bullet points 
Option 1: Configure Zowe manually using the `zwe init` command group 
  To run the `zwe init` command, it is necessary to create a <...> 
Option 2: Configure Zowe with z/OSMF workflows
  You can execute the Zowe configuration workflow <...>

File: docs/user-guide/configure-zowe-runtime.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Initializing Zowe z/OS runtime
```
Comment (skurnevich) [inline docs/user-guide/configure-zowe-runtime.md:1]: I believe this page is redundant and could be removed entirely for the simplicity

File: docs/user-guide/zwe-init-subcommand-overview.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
+# zwe init subcommand overview
+
+Review this article to learn about the individual subcommands executed in `zwe init`. Based on your use case, you may choose to run the subcommands of `zwe init` individually rather than running all of these commands together. Review this article to get started with using `zwe init` subcommands.
+
+:::important
+Some of the following `zwe init` subcommands require elevated permissions. See the required roles associated with each of these commands.
+:::
+
+* [Initializing Zowe custom data sets (`zwe init mvs`)](#initializing-zowe-custom-data-sets-zwe-init-mvs)
+* [Initializing Zowe security configurations  (`zwe init security`)](#initializing-zowe-security-configurations-zwe-init-security)
```
Comment (skurnevich) [inline docs/user-guide/zwe-init-subcommand-overview.md:10]: This anchor link is not working for me

File: docs/user-guide/zwe-init-subcommand-overview.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+This subcommand creates the user IDs and security manager settings.
+
+:::info**Required role:** security administrator
+:::
+
+If Zowe has already been launched on a z/OS system from a previous release of Zowe v2, you can skip this security configuration step unless told otherwise in the release documentation.
+
+The JCL member `.SZWESAMP(ZWESECUR)` is provided to assist with the security configuration. Before submitting the `ZWESECUR` JCL member, customize this member to match site security rules. For script driven scenarios, you can run the command `zwe init security` which uses `ZWESECUR` as a template to create a customized member in `.CUST.JCLLIB`.  This member contains the commands required to perform the security configuration. 
+
+For more information about `zwe init security`, see [Initializing Zowe security configurations](./initialize-security-configuration.).
```
Comment (skurnevich) [inline docs/user-guide/zwe-init-subcommand-overview.md:112]: Broken link - extra dot in the end

File: docs/user-guide/zwe-init-subcommand-overview.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+## Creating VSAM caching service datasets (`zwe init vsam`)
+
+Zowe can work in a high availability (HA) configuration where multiple instances of the Zowe launcher are started, either on the same LPAR or different LPARs connected through sysplex distributor. If you are only running a single Zowe instance on a single LPAR you do not need to create a caching service so you may skip this step.
+
+:::info**Required roles:** system programmer
+:::
+
+The command `zwe init vsam` uses the template JCL in `SZWESAMP(ZWECSVSM)` to copy the source template member from `zowe.setup.mvs.hlq.SZWESAMP(ZWECVCSM)` and creates a target JCL member in `zowe.setup.mvs.jcllib(ZWECVSCM)` with values extracted from the `zowe.yaml` file.
+
+For more information about `zwe init vsam`, see [Creating VSAM caching service datasets](./initialize-vsam-dataset).
```
Comment (skurnevich) [inline docs/user-guide/zwe-init-subcommand-overview.md:151]: Link leads to old VSAM page, should point to section of configure-caching-service-ha

File: docs/user-guide/zwe-init-subcommand-overview.md [hunk 6]
```diff
@@ -0,0 +1,202 @@
...
+>> Zowe main started tasks are installed successfully.
+#>
+```
+
+## Next steps
+
+After each of the `zwe init` subcommands run successfully, and you completed the security configuration, assigned security permissions to users, and configured the cross memory server (ZIS), you can procede to [starting Zowe](./start-zowe-zos.md).
+
+* For more information about configuring security, see the articles in the section [Configuring security](./configuring-security).
+* For more information about configuring the cross memory server (ZIS), see [Configuring the Zowe cross memory server (ZIS)](./configure-xmem-server.md).
```
Comment (skurnevich) [inline docs/user-guide/zwe-init-subcommand-overview.md:202]: Major parts of the security and cross memory server configurations is not done by zwe init and so it is to early to proceed to starting Zowe yet, at least swap these lines to first propose to go to security configuration and ZIS config

File: docs/user-guide/configure-zowe-runtime.md (hunk 2 above)
Comment (janan07) [reply docs/user-guide/configure-zowe-runtime.md:1]: The purpose of this article is that there are two options under this category (config with zwe init, and z/OSMF workflow. I see your point that it is a bit redundant, but it does help to keep the options discrete under a single category.

**Pull request merged**